src/main/drivers/bus_spi_impl.h

```diff
@@ -20,6 +20,9 @@
 
 #pragma once
 
+#define SPI_TIMEOUT_US          10000
+#define SPI_TIMEOUT_SYS_TICKS   (SPI_TIMEOUT_US / 1000)
```

Comment:
Is this used anywhere outside of `src/main/drivers/bus_spi_hal.c`? If not then it should probably live in there.

Reply:
No it's not. Moved it 👍